docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -1,42 +1,67 @@
+Devfiles provide guidelines and instructions for enhanced development processes. Devfiles uses the command line interface (CLI)  tool, odo, to execute the actions: build run, deploy. By using odo, devfiles provide consistent patterns of behavior for applications written in different languages and create workspaces composed of multiple containers. In these containers, you can make any number of identical workspaces from the same devfile. Creating multiple workspaces lets you share your devfile with different teams. Sharing a single devfile across multiple teams, ensures that each team has the same user experience and the same build, run, deploy behaviors.
+
+Leverage the following devfile features:
+
+* Guidance for using runtime images
 * Example code
 * Build and CI commands
 * Deployment options
 
-Benefits of devfiles:
+Use devfiles to help you with the following:
+
+* Reduce the gap between development and deployment
+* Have a registry for finding and hosting stacks
+* Produce consistent build and run behaviors
+* Allow easy devfile customization
 
-* Reducing the gap between development and deployment
-* Having a registry for finding and hosting devfiles
-* Producing consistent build and run behaviors
-* Allowing easy devfile customization
+For a visual of how simple a devfile is, see the following YAML:
 
-.Who can use devfiles?
+.A minimal devfile with one single project
+====
+[source,yaml]
+----
+schemaVersion: 2.1.0
+metadata:
+  name: petclinic-dev-environment
+  version: 1.0.0
+projects:
+  - name: petclinic
+    git:
+      remotes:
+        origin: "https://github.com/spring-projects/spring-petclinic.git"
+      checkoutFrom:
+        revision: main
+----
+====
+
+.Roles that use devfiles
 
 Stack providers use devfiles to develop their runtime stacks once and then use these runtime stacks across different tools. Therefore, with devfiles, stack providers do not need to build and maintain custom development tools.
 
-Tool providers use devfiles to build runtime support for each runtime. Now, the devfile runtime teams build and run applications on their servers, giving tool providers more time.
+Tool providers use devfiles to build runtime support for each runtime. The devfile runtime teams build and run applications on their servers, giving tool providers more time.
+
+With devfiles, developers do not need to set up the environment for building and running their applications. Instead, developers have the flexibility to use different tools, like Che and odo.
+
+.How to work with devfiles
 
-Developers use devfiles to focus only on developing their applications. With devfiles, developers do not need to set up the environment for building and running their applications. Instead, developers have the flexibility to use different tools, like Che and odo.
+* To access the devfile stacks and images so you can begin creating container workspaces, see the link:https://github.com/devfile/registry/tree/main/stacks[devfile registry stacks].
```

Comment:
Articles in front of the links work only for the first two bullet points, and also only if you add the description in the square brackets. That looks better than having .adoc at the end anyway, doesn't it? 
You could keep the articles for the last two points by adding "guide". For example: see the [Migrating to Devfile v2] guide.

Reply:
I removed the articles `the` before the links. Here's how the links render. There's no `adoc` in the rendered version: 
<img width="881" alt="Screen Shot 2021-07-06 at 3 04 58 PM" src="https://user-images.githubusercontent.com/70717303/124653715-a1105080-de6b-11eb-9d77-05f31f656fe8.png">